### PR TITLE
Fixes for the Concepts docs

### PR DESCRIPTION
**Pull request by fhs-improving**

### CODE REVIEW

File: doc/src/hugo/content/concepts/processor.md [hunk 1]
```diff
@@ -4,9 +4,9 @@ draft: false
 ---
 
 A processor is a component of any [vital definition]({{< relref "vital" >}}). 
-Processors perform some transformation, typically, of the data flowing in from
```
Comment (reid-spencer) [inline doc/src/hugo/content/concepts/processor.md:7]: I think the word "typically" is useless here. The only transformation processors should do is of their input from inlets. So the typical case is the always cas too. Could you just remove "typically" from that sentence, please?

Comment (reid-spencer) [reply doc/src/hugo/content/concepts/processor.md:7]: Same thing with "potentially diffferent" below.  If there was no transformation, there would be no need to have a processor at all.

**Review by reid-spencer — CHANGES_REQUESTED** [review]
Thanks for catching all those typos.  I've requested correction of a couple thinkos (on my part) too.  Nice finds!

File: doc/src/hugo/content/concepts/processor.md (hunk 1 above)
Comment (fhs-improving) [reply doc/src/hugo/content/concepts/processor.md:7]: Should it be "produces some different data" or just "produces some data"?

Comment (reid-spencer) [reply doc/src/hugo/content/concepts/processor.md:7]: _A processor performs some transformation of the data flowing in from its [inlet](s) and produces different data to its [outlet](s)._

**Review by reid-spencer — CHANGES_REQUESTED** [review]

**Comment by reid-spencer** [comment]
@fhs-improving - Don't forget to pull before you push because this branch has had updates from dependency updates.

File: doc/src/hugo/content/concepts/processor.md (hunk 1 above)
Comment (fhs-improving) [reply doc/src/hugo/content/concepts/processor.md:7]: fixed!

**Review by reid-spencer — APPROVED** [review]
LGTM

**Pull request merged**